oneview_redfish_toolkit/api/redfish_json_validator.py

```diff
@@ -57,22 +59,37 @@ def _validate(self):
         """Validates self.redfish against self.schema_obj
 
             Validates a redfish OrderedDict against the schema object passed
-            on the object creation
+            on the object creation.
 
             Returns:
                 None
 
             Exception:
                 raises an exception on validation failure
         """
+        schema_dir = util.config['redfish']['schema_dir']
+        schema_paths = glob.glob(os.getcwd() + '/' + schema_dir + '/*.json')
+
+        store = {}
```

Comment:
Good! You are building a local store for the RefResolver... not sure if redfish_json_validator is the best place to load this info. This way, you'll end up loading/reloading all schema_dir files every time you instantiate an object.

Would utils be a better place to load this? We could add to globals and just import here...

Reply:
Great, I'll fix it.